source/test/E2E.test.js

```diff
@@ -0,0 +1,98 @@
+
+describe('Basic web functionality check', () => {
+
+    jest.setTimeout(50000);
+
+
+
+    it('Check thumnails on home page',async () => {
+        const puppeteer = require('puppeteer');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8436700)
```suggestion
        const puppeteer = require("puppeteer");
```